src/sfctl/custom_chaos_schedule.py

```diff
@@ -0,0 +1,163 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom commands for the Service Fabric chaos schedule test service"""
+
+def parse_time_of_day(time_of_day):
+    """Parse a TimeOfDay from string"""
```

Comment:
Is time_of_day here a string or a dictionary?

Reply:
time of day is 
{
  "Hour": int,
  "Minute": int
}